plugin/pkg/admission/imagepolicy/admission_test.go

```diff
@@ -1057,7 +1057,10 @@ func TestReturnedAnnotationAdd(t *testing.T) {
 			annotations := make(map[string]string)
 			attr = &fakeAttributes{attr, annotations}
 
-			err = wh.Validate(context.TODO(), attr, nil)
+			if err := wh.Validate(context.TODO(), attr, nil); err != nil {
```

Comment:
err is expected to be non-nil for `test: "Adds annotations for bad request",`... need to indicate that in the testcase

Reply:
thanks for your guide. I will do it later.